src/interfaces/index.ts

```diff
@@ -30,6 +28,7 @@ export * from './filing-data-interface'
 export * from './dissolution-filing-interface'
 export * from './form-interface'
 export * from './history-item-interface'
+export * from './name-request-interface'
```

Comment:
Now importing NR interfaces locally instead of from shared library (where they are the "UI" object not the actual "NR" objects).